src/LiveComponent/src/Util/LiveControllerAttributesCreator.php

```diff
@@ -98,6 +98,13 @@ public function attributesForRendering(MountedComponent $mounted, ComponentMetad
             $mountedAttributes = $mountedAttributes->defaults(['data-live-id' => $id]);
         }
 
+        $forcePost = AsLiveComponent::forcePost($mounted->getComponent());
```

Comment:
Is this specific case needed ? Is it possible to use `AsLiveComponent::serviceConfig()` like for 'route' ?

Reply:
That's totally possible, I'll refactor this part to use `serviceConfig()`.

Reply:
I've added 'force_post' to `serviceConfig()`, but I also kept the `forcePost()` helper method. The `serviceConfig()` is not easily accessible in `attributesForRendering()`.